packages/teams-js/test/public/pages.spec.ts

```diff
@@ -89,52 +116,42 @@ describe('AppSDK-TeamsAPIs', () => {
     ];
 
     it('should not allow calls before initialization', async () => {
-      expect.assertions(1);
-      await pages
-        .navigateCrossDomain('https://valid.origin.com')
-        .catch(e => expect(e).toMatchObject(new Error('The library has not yet been initialized')));
+      await expect(pages.navigateCrossDomain('https://valid.origin.com')).rejects.toThrowError(
+        'The library has not yet been initialized',
+      );
     });
 
-    it('should not allow calls with a bad origin', async () => {
-      expect.assertions(1);
-      await pages
-        .navigateCrossDomain('https://badorigin.com')
-        .catch(e => expect(e).toMatchObject(new Error('The library has not yet been initialized')));
+    // Skipping these tests as url validation is not implemented
```

Comment:
Isn't this test testing the same thing as the previous one?

Reply:
I see, so I think you converted tests that were already here and in place. In my opinion, might be better to comment out the tests for features that don't work yet and then open bugs for this work if there aren't already. We can then maybe add the bug number(s) in the comments here?

Reply:
Sounds good. I commented out the tests and created bug 5790202 for this issue